gradio-notebook/README-dev.md

```diff
@@ -17,7 +17,6 @@ Just a heads up, the process for getting local development setup for the first t
 ### Setting up Gradio repo
 
 1. Go to the project where this is defined. Ex: `~/Projects/gradio-workbook`
-2. Delete this entire repo! `rm -rf gradioworkbook`
```

Comment:
nit: Adjust the numbers below. Even though I know Markdown will increment properly just easier to follow